qiskit/circuit/quantumcircuit.py

```diff
@@ -3115,6 +3117,13 @@ def decompose(
         # do not copy operations, this is done in the conversion with circuit_to_dag
         return dag_to_circuit(dag, copy_operations=False)
 
+    @deprecate_arg(
+        "justify",
+        since="1.1.0",
+        deprecation_description="Setting QuantumCircuit.draw()’s justify argument to a value other than "
+        "'left', 'right', or 'none'. Until deprecation, default 'left' will be used instead.",
+        predicate=lambda justify: justify not in ["left", "right", "none"],
```

Comment:
```suggestion
        predicate=_is_valid_justify_arg,
```

Reply:
Obviously! My bad, adding it :+1:

Reply:
In the end, I went back to rewriting the lambda function in the `@deprecate_arg`, instead of importing it, due to circular import errors it produces...

And since it's just a temporary one-liner, I hope having duplicated code there is not too bad... If not please tell me 👌

Reply:
**Update, from call with luciano and Kevin:**

They think in this case the wrapper complicates things instead of helping, so we should maybe remove it 👍